### PR TITLE
Issu01 Remove reduce source code

### PR DESCRIPTION
**Pull request by thientt**
Hi @tranthiencdsp 

I done have remove using reduce in sourcecode
Pnease you can review now.

Thanks
ThienTran

### CODE REVIEW

File: ConsoleAppWithGitHub/ConsoleApp/ConsoleApp/ISongMucis.cs [hunk 1]
```diff
@@ -1,10 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace MockingWithIoC
+﻿namespace MockingWithIoC
```
Comment (tranthiencdsp) [inline ConsoleAppWithGitHub/ConsoleApp/ConsoleApp/ISongMucis.cs:1]: OK

File: ConsoleAppWithGitHub/ConsoleApp/ConsoleApp/Program.cs [hunk 2]
```diff
@@ -1,33 +1,17 @@
...
-            DateTime now = DateTime.Now;
-            Console.WriteLine(now.ToString());
-            CultureInfo info = CultureInfo.GetCultureInfo("vi-VN");
-            CultureInfo.DefaultThreadCurrentCulture = info;
-            CultureInfo.DefaultThreadCurrentUICulture = info;
-            CultureInfo infoJP = CultureInfo.GetCultureInfo(1041);
-            Console.WriteLine(infoJP.TextInfo.ANSICodePage.ToString());
-            Console.WriteLine(now.ToString(infoJP));
+            string jsonStr = CreateStateJsonString();
+            var persons = jsonStr.DeserializeArraySafe<Person>();
```
Comment (tranthiencdsp) [inline ConsoleAppWithGitHub/ConsoleApp/ConsoleApp/Program.cs:14]: We should use Person[] of instead var for in the case.

**Review by tranthiencdsp — COMMENTED** [review]
Done

File: ConsoleAppWithGitHub/ConsoleApp/ConsoleApp/ISongMucis.cs (hunk 1 above)
Comment (thientt) [reply ConsoleAppWithGitHub/ConsoleApp/ConsoleApp/ISongMucis.cs:1]: Hi Thien

Thanks you were review source code for me.
Thanks
TT

File: ConsoleAppWithGitHub/ConsoleApp/ConsoleApp/Program.cs (hunk 2 above)
Comment (thientt) [reply ConsoleAppWithGitHub/ConsoleApp/ConsoleApp/Program.cs:14]: Hi T

I will change source as your comment
Thanks

**Pull request merged**